dataland-frontend/src/components/general/JoinDatalandButton.vue

```diff
@@ -1,14 +1,12 @@
 <template>
-  <div class="col-12">
```

Comment:
Note for myself check the page layout once the fallback for the sample page is implemented